src/components/common/Header.js

```diff
@@ -1,6 +1,10 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import { isAuthenticated, logout } from "../../helper/LoginHelper";
+import { logout } from "../../helper/LoginHelper";
+
+const rightStyle = {
+  float: 'right'
+}
```

Comment:
put this piece in index.css and call it here.

Reply:
If you check the index.css, the default style for nav li is float left. This is for overriding the default style for one li element, which contains the Log Out link.

Reply:
If you check the index.css, the default style for nav li is float left. This is for overriding the default style for one li element, which contains the Log Out link.